File: front/customer_app/lib/models/item.dart
import 'package:flutter/services.dart';

class Item {
  late String id;
  late String categoryId;
  late String name;
  late String description;
  late double price;
  late bool isSold;
  late String image;
  Item(
      {required this.id,
      required this.categoryId,
      required this.name,
      required this.description,
      required this.price,
      required this.isSold,
      required this.image});

  Item.fromJson(Map<String, dynamic> json)
      : id = json['Id'],
        categoryId = json['CategoryId'],
        name = json['Name'],
        description = json['Desciption'],
        price = json['Price'],
        isSold = json['IsSold'],
        image = json['Image'];
}

File: back/MobileApi/MobileApi/Models/Item.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace MobileApi.Models
{
    public partial class Item
    {
        public Item()
        {
            CartToItems = new HashSet<CartToItem>();
        }

        public Guid Id { get; set; }
        public Guid CategoryId { get; set; }
        public string Name { get; set; }
        public string Desciption { get; set; }
        public decimal Price { get; set; }
        public bool IsSold { get; set; }
        public byte[] Image { get; set; }

        public virtual Category Category { get; set; }
        public virtual ICollection<CartToItem> CartToItems { get; set; }
    }
}
